README.md

```diff
@@ -0,0 +1,25 @@
+# Delete duplicate module records
+
+The module record table in the learner_record database currently doesn't have uniqueness on module_id and user_id. This can cause errors loading completion status of some courses.
+
+## Running
+
+### Requirements
+
+Requirements for this script can be found in `requirements.txt`
+
```

Comment:
Might be worth including an instruction to install the requirements with `pip install -r requirements.txt`